integration/examples/ruby/skaffold.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: skaffold/v2alpha1
+kind: Config
+build:
+  artifacts:
+  - image: ruby-example
+    context: backend
+    sync:
```

Comment:
Have you tried?

```
sync:
  infer:
  - *.rb
```

It would be nice to demonstrate inferred sync

Reply:
Yep, I've changed to rack unreloader config and skaffold's option to infer. Works well